lib/rules/color-hex-alpha/index.js

```diff
@@ -0,0 +1,69 @@
+// @ts-nocheck
+'use strict';
+
+const declarationValueIndex = require('../../utils/declarationValueIndex');
+const isStandardSyntaxDeclaration = require('../../utils/isStandardSyntaxDeclaration');
+const isStandardSyntaxValue = require('../../utils/isStandardSyntaxValue');
+const report = require('../../utils/report');
+const ruleMessages = require('../../utils/ruleMessages');
+const validateOptions = require('../../utils/validateOptions');
+const valueParser = require('postcss-value-parser');
+const ruleName = 'color-hex-alpha';
+
+const messages = ruleMessages(ruleName, {
+	expected: (unfixed) => `Expected  alpha channel in "${unfixed}"`,
+	unexpected: (unfixed) => `Unexpected  alpha channel in "${unfixed}"`,
+});
+
+function rule(primary) {
+	return (root, result) => {
+		const validOptions = validateOptions(result, ruleName, {
+			actual: primary,
+			possible: ['always', 'never'],
+		});
+
+		if (!validOptions) return;
+
+		root.walkDecls((decl) => {
+			if (!isStandardSyntaxDeclaration(decl)) return;
+
+			const parsedValue = valueParser(getValue(decl));
```

Comment:
```suggestion
			const parsedValue = valueParser(decl.value));
```

`decl.value` will suffice as we're not changing the AST.